add basic polymer youtube player

eleanor-byhook opened the pull request:
controllers hidden
size is fluid (100% of browser)
beginning and end frame are the same

eleanor-byhook committed:
add basic polymer youtube player

controllers hidden
size is fluid (100% of browser)
beginning and end frame are the same

JeremiahReichardt commented:
Looks good, can you pull this player into an `elements/page-youtube-player/page-youtube-player.html` file. Add in the `index.html` with the `iron-pages` element as well as the `demo` folder. Please removes changes to the global styles these should be inside your own element.

eleanor-byhook commented:
Can do

eleanor-byhook committed:
PR feedback

wrapped google-youtube player in page-youtube-player element
added index.html and demo.html page for the page-youtube-player element
added JS documentation for page-youtube-player
removed changes to global styles into element
